Makefile

```diff
@@ -237,18 +237,6 @@ scan:
 		--ignore-unfixed \
 		./
 
-.PHONY: lint
```

Comment:
Wasn't the aim not to run golangci-lint in CI? I think we want to keep this, for manually running it.

Reply:
the message we got was that "Org wide, we are not recommending or using golangci-lint".  As per this, I've removed it entirely.  The results aren't that useful to us.